smbios.adoc

```diff
@@ -22,7 +22,51 @@ on top of cite:[SMBIOS], and is optional and recommended for BRS-B. Additional n
 | 41   | Onboard Devices Extended Information      | Recommended |
 | 42   | Redfish Host Interface                    | Conditional | Required when Redfish host interface present.
 | 43   | TPM Device                                | Conditional | Required when TPM present.
-| 44   | Standard Processor Additional Information | Required    |
+| 44   | Processor Additional Information | Required    |
 | 45   | Firwmare Inventory Information            | Recommended |
 | 46   | String Property                           | Recommended |
 |===
+
+=== Type 44 Processor-Specific Data
+
+The processor-specific data structure fields are defined to follow the standard Processor-Specific Block fields (cite:[SMBIOS], Section 7.45.1).
+
+The structure is defined in a manner consistent with the DMTF specification
+language (cite:[SMBIOS]), and is valid for processors declared as
+architecture 07h (64-bit RISC-V) only.
+
+[cols="2,2,3,2,2,4", width=95%, align="center", options="header"]
+|===
+| Offset | Version | Name      | Length | Value   | Description
+| 00h| 0100h|Revision|WORD|Varies|See <<smbios-psd-ver>>.
+| 02h| 0100h| Hart ID| QWORD| Varies| The ID of this RISC-V hart
+| 0Ah| 0100h| Boot hart| BYTE| Boolean| 1: This is the boot hart. +
+0: This is not the boot hart.
+| 0Bh| 0100h| Machine Vendor ID | QWORD| Varies| The vendor ID of this
+RISC-V hart
+| 13h| 0100h| Machine Architecture ID| QWORD| Varies| Base
+microarchitecture of the hart. Value of 0 is possible to indicate the field is
+not implemented. The combination of Machine Architecture ID and Machine Vendor
+ID should uniquely identify the type of hart microarchitecture that is implemented.
+| 1Bh| 0100h| Machine Implementation ID| QWORD| Varies| Unique encoding
```

Comment:
While I understand this is a spec, I have an implementation question for these fields. These vendor ID/impl ID for each hart can be fetched using SBI call from S-mode FW only from the hart which executes it right? Typically, the SMBIOS creator FW like EDK2 runs with single CPU. How is it supposed to create these fields  each hart?